### PR TITLE
Removing no_rocm tag from XLA unit tests that pass with ROCm 2.4

### PR DESCRIPTION
**Pull request by deven-amd**

### CODE REVIEW

File: tensorflow/compiler/jit/xla_gpu_device.cc [hunk 1]
```diff
@@ -103,7 +106,8 @@ Status XlaGpuDeviceFactory::CreateDevices(
   (void)registrations;
 
   // XXX FIXME devise a way to cope with multiple platforms
```
Comment (whchung) [inline tensorflow/compiler/jit/xla_gpu_device.cc]: remove "XXX FIXME"?

Comment (deven-amd) [reply tensorflow/compiler/jit/xla_gpu_device.cc]: will do.

either in this PR (if CI fails and I need to push out another commit) or in the next PR I file

**Comment by deven-amd** [comment]
`rocm`, `rocm-v2`, and `rocm-xla` all passed, an I am unable to reproduce the cuda test failure locally. I randomly picked 3 of the many cuda failures, and reran them locally...they all passed. So guessing this is a CI node issue.

will issue a retest command after this post...we only need to check for the `cuda` path in the retest

**Comment by deven-amd** [comment]
jenkins : retest this please

**Comment by deven-amd** [comment]
jenkins : retest this please

**Comment by deven-amd** [comment]
@whchung @parallelo @sunway513 

I am stumped on this one.

I consistently get the failures on the `cuda` CI run with this PR. The failures are the same everytime (have run this 3 times so far), but I cannot reproduce them locally. Moreover all the changes here are on the XLA side of the fence, and `cuda` CI run is for the non-XLA tests, so do not know what is going on.

Any tips on how to reproduce this error locally, or what could be causing this failure?

thanks

**Comment by deven-amd** [comment]
btw, the tests all fail with the following error:

```
2019-05-13 16:22:57.344810: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:1005] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2019-05-13 16:22:57.348592: F tensorflow/stream_executor/lib/statusor.cc:34] Attempting to fetch value instead of handling error Not found: could not find registered transfer manager for platform CUDA -- check target linkage
Fatal Python error: Aborted
```

**Comment by whchung** [comment]
https://stackoverflow.com/questions/36838770/how-to-interpret-tensorflow-output
https://stackoverflow.com/questions/44232898/memoryerror-in-tensorflow-and-successful-numa-node-read-from-sysfs-had-negativ

looks like an environment issue.

**Comment by deven-amd** [comment]
but `cuda` CI is passing for other PRs and consistenly failing only this PR...so something in this PR is causing it to fail

**Comment by whchung** [comment]
https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/blob/develop-upstream-deven-190510-misc/tensorflow/compiler/xla/service/gpu/gpu_transfer_manager.cc#L199

based on the error code on `cuda`, this `XXX` might need to be addressed.

**Comment by whchung** [comment]
the message `returning NUMA node zero` should be benign. the root cause should be transfer manager for CUDA not registered.

**Comment by deven-amd** [comment]
found a potential cause of the error :

https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/blob/develop-upstream/tensorflow/compiler/xla/service/gpu/gpu_transfer_manager.cc#L201

only trouble is that should cause a failure even without this PR.

My guess is that the following change (in this PR) is somehow triggering the error
https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/pull/450/files#diff-044f7b34d4067f1bbdf218d11fa26096R66

We had the platform name hardcoded to "CUDA" (and somehow that was working for ROCm). Fixing that is maybe triggering the error I am running into (on just the) CI nodes.

I will push out a commit shortly to "fix" the code in `gpu_transfer_manager.cc`....hoping CI takes a liking to it.

**Comment by deven-amd** [comment]
@whchung just saw your message...should have read it earlier....guess we both reached the same spot!

**Pull request merged**